docs/UserGuide.md

```diff
@@ -74,67 +71,62 @@ can effortlessly manage their finances and social engagements.
-
     * `exit` : Exits the app.
+6. **Command Overview:** Need more details on each command? Check out the [Command Overview](#command-overview) below.
 
-6. Refer to the [Command Overview](#command-overview) below for details of each command.
-
---------------------------------------------------------------------------------------------------------------------
+---
 
 ## Features
 
 ### User Interface Overview
 
-When you launch FriendFolio, you will be greeted with some key information on the dashboard. Let's walk you through some
-of the components in the dashboard!
+When you launch FriendFolio, you will be greeted with some key information on the dashboard. Let's take a tour of what you'll find when you launch the app.
 
 ![Breakdown of Ui](images/UiBreakdown.png)
 
 **Dashboard:**
 
-The dashboard gives you an overview of your day. Here are the components of the dashboard.
+Get ready for a quick glance at your day! Here's what you'll find on the dashboard:
 
-* The dashboard displays the current time so that you do not have to dart your eyes elsewhere to find out what time it
-  currently is.
-* FriendFolio tells you the number of contacts you have so that you have a sensing of how many contacts you have. The
-  more, the merrier!
-* There is a graph to show you how much you are owed, and how much you owe others. At one glance, you can find out if
-  you need to transfer anyone money, or if you need to chase anyone for payment.
-* FriendFolio tells you who is available today, so you know if your friends are free to meet you for a meal, or if they
-  are free to discuss that one group project you all might have been procrastinating on.
+* **Current Time:** Stay on track with the current time displayed right here, so you don't have to look away.
+* **Contact Count:** See how many buddies you've got in your network because the more, the merrier!
+* **Finances Graph:** Keep tabs on who owes you and who you owe, all in one neat graph. No more guesswork on who needs a gentle reminder about that borrowed cash.
+* **Availability Status:** Know at a glance who's free today, perfect for planning catch-ups or tackling group projects together.
 
-**Command Line Interface:**
+**Command Box:**
 
-* At the top of the screen is the command line interface. You can type in commands into the command box and the command
-  results will be displayed in the box above!
+* At the top of the screen is our trusty command box where you can type commands into. The command results will be displayed in the box above after hitting Enter.
 
 **Contact List:**
 
-* On the left is where you can see your contacts. Clicking on these cards will replace the dashboard with more detailed
-  information the contact that you clicked on. You can unselect the contact by pressing on the `Esc` key.
+* On the left is where you can see your contacts. Click on any card to dive into more details about that friend. And don't worry, if you want to go back, just hit `Esc`.
```

Comment:
perhaps the part on if you want to go back can use if you want to go back to the dashboard, just hit `Esc`. (Because I realize there's no mention of dashboard on this line) Just to be clear.